microsoftgraph/microsoft-graph-explorer-v4
# Task: Make request headers editable

## Overview
Fixes #1516 

### Demo
<img width="1034" alt="image" src="https://user-images.githubusercontent.com/45680252/171596985-17bcbaf1-691f-4646-ba8a-8fa515dd4f0b.png">

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
Check out this branch and run it locally
Add a request header
Notice that a new action (edit) is present, which allows you to edit the header
